src/krb5/_set_password.pyi

```diff
@@ -0,0 +1,99 @@
+import typing
+
+from krb5._ccache import CCache
+from krb5._context import Context
+from krb5._creds import Creds
+from krb5._principal import Principal
+
+class SetPasswordResult(typing.NamedTuple):
+    result_code: int
```

Comment:
You can add the comments to each result here in the pyi instead of repeating it on every Returns field. For example

```python
class SetPasswordResult(typing.NamedTuple):
    """Results for set_password calls.

    This is the result returned by set_password and set_password_using_ccache.
    Blah blah blah.
    """

    result_code: int
    """More info about this."""
    result_code_string: bytes
    """Foo."""
    result_string: str
    """Bar."""
```

With this you can just say in each function

```python
"""
Returns:
    SetPasswordResult: See SetPasswordResult for more
    information about the return result.
"""
```

Reply:
done